subsys/bluetooth/audio/mpl_internal.h

```diff
@@ -10,54 +10,53 @@
 #ifndef ZEPHYR_SUBSYS_BLUETOOTH_AUDIO_MPL_INTERNAL_
 #define ZEPHYR_SUBSYS_BLUETOOTH_AUDIO_MPL_INTERNAL_
 
-#include <bluetooth/audio/media_proxy.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-
 /* Offset into a segment/track before the "previous" command goes to start of */
 /* current segment/track instead to previous */
 #define PREV_MARGIN           500  /* 500 * 0.01 = 5 seconds */
 
 /* Increase/decrease in seeking sped factor for fast rewind/forward commands
+ * The media control specification has a requirement that the speed factor
+ * shall be [-64, -4], 0, [4, 64].  I.e., values between 0 and +/- 4 are not allowed.
  * Set this equal to the minimum speed factor, to ensure only valid speed factors
  * are used when changing to/from zero
  */
-#define MPL_SEEKING_SPEED_FACTOR_STEP  BT_MCS_SEEKING_SPEED_FACTOR_MIN
+#define MPL_SEEKING_SPEED_FACTOR_STEP  4
```

Comment:
Should this actually have been a Kconfig?

Reply:
This is left unspecified (on purpose) in the Media Control Service specification.  And it is likely to be very implementation specific.  So I see no value in making it configurable for this sample player.

Reply:
Everything related to the MPL is implementation specific, and yet we have plenty of Kconfigs for it already (including the player name, etc.), so I do not follow your argument. 

However, given that the MPL will be moved to `samples`, then Kconfigs are not necessary in any case.

Reply:
I think the argument is more or less like this: The MCS spec says that there shall be a media player name (in the service).  So it makes sense to have a name for the player.  But for the seeking speed behaviour, the MCS deliberately leaves almost everything up to the implementation.

Reply:
The media player name, and the media player seeking speed factor are both 2 values that reside in the media player and are implementation specific. I don't see why they should be treated different in any way. Whether they are controlled via Kconfigs or hardcoded in header files doesn't really matter once this is moved to a sample application.